src/backend/distributed/planner/multi_physical_planner.c

```diff
@@ -2821,6 +2825,50 @@ SqlTaskList(Job *job)
 }
 
 
+/*
+ * RelabelTypeMutator walks over the tree and converts RelabelType's
+ * into CollationExpr's. With that, we will be able to pushdown COLLATE's.
+ */
+static Node *
+RelabelTypeMutator(Node *originalNode)
```

Comment:
maybe `RelabelTypeMutator` -> `ReplaceRelabelTypesWithCollationExprs`

Also, I think that part of the comment `With that, we will be able to pushdown COLLATE's.` should go into where we call this function

```c
/* to pushdown collate's, we need to convert RelabelType to CollationExpr */
jobQuery = RelabelTypeMutator(jobQuery);
```